src/Evolution/Executables/RelativisticEuler/Valencia/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBS_TO_LINK
+  CoordinateMaps
+  DiscontinuousGalerkin
+  DomainCreators
+  GeneralRelativitySolutions
+  Hydro
+  IO
+  Informer
+  Limiters
+  LinearOperators
+  MathFunctions
+  RelativisticEulerSolutions
+  Time
+  Utilities
+  Valencia
+  )
+
+function(add_valencia_executable EXECUTABLE_NAME DIM INITIAL_DATA)
+  add_spectre_parallel_executable(
+    "Evolve${EXECUTABLE_NAME}${DIM}D"
+    EvolveValencia
+    Evolution/Executables/RelativisticEuler/Valencia
+    "EvolutionMetavars<${DIM},${INITIAL_DATA}>"
+    "${LIBS_TO_LINK}"
+    )
+endfunction(add_valencia_executable)
+
+function(add_smooth_flow_executable DIM)
+  add_valencia_executable(
+    SmoothFlow
```

Comment:
We already have an "EvolveSmoothFlow" executable for GRMHD.  This adds "EvolveSmoothFlow1D", etc., so it doesn't technically conflict, but it is kind of confusing.  We should pick some way of disambiguating these generic solutions for different systems.  Perhaps call these "EvolveRelativisticEulerSmoothFlow1D" or something and then we can rename the other one to "EvolveGrmhdSmoothFlow"?

Reply:
Any other suggestions for naming these? If not, I believe `EvolveValenciaSmoothFlow1D` is more accurate (since the executable is compiled with the Valencia formulation)

Reply:
Seems reasonable to me

Reply:
Sure, that seems fine.